include/stdio.h

```diff
@@ -41,19 +42,17 @@ typedef struct FILE FILE;
 
 struct FILE {
 	int fd;		/* UNIX file pointer */
-	char flags;	/* bits for must free buffer on close, line-buffered */
+	char p9flags;	/* bits for must free buffer on close, line-buffered */
```

Comment:
huh? this is old, now FILE struct is not in stdio, just a typedef _IO_FILE FILE and moved to internal stdio_impl.h. And that part is gone. What version are you looking?